apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/VilkaarsvurderingBehandlingDTO.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.etterlatte.vilkaarsvurdering
+
+import no.nav.etterlatte.libs.common.behandling.BehandlingType
+import no.nav.etterlatte.libs.common.behandling.DetaljertBehandling
+import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.behandling.Virkningstidspunkt
+
+data class VilkaarsvurderingBehandlingDTO(
+    val sak: Long,
+    val sakType: SakType,
+    val behandlingType: BehandlingType,
+    val virkningstidspunkt: Virkningstidspunkt?
+) {
```

Comment:
Er den nødvendig å ha? Ser ut som et subset av `DetaljertBehandling`.

Reply:
Oi, det har du rett i. Hadde bruk for denne da eg prøvde ein annan variant undervegs, men no har den jo kun data som alt fins i DetaljertBehandling, og da er det jo ikkje vits lell.